packages/gatsby-plugin-image/src/transforms/__testfixtures__/gatsby-plugin-image/uses-src.output.js

```diff
@@ -0,0 +1,18 @@
+import { GatsbyImage } from "gatsby-plugin-image";
+
+<GatsbyImage
+  image={{
+    srcSet: ``,
+    src:
```

Comment:
I'm pretty sure this won't work. The image object shape is different.

Reply:
Can you elaborate? This was taken from a real example site.